examples/siro_sandbox/server/remote_gui_input.py

```diff
@@ -260,3 +260,6 @@ def update(self):
 
     def on_frame_end(self):
         self._gui_input.on_frame_end()
+
+    def on_reset(self):
```

Comment:
`on_reset` is an odd name. I wouldn't expect a low-level input class like this to know about RL concepts like episode reset. How about just `clear_history`.